content/docs/observability/otel-stack.md

```diff
@@ -0,0 +1,704 @@
+   ingester:
+     replicas: 0
+   querier:
+     replicas: 0
+   queryFrontend:
+     replicas: 0
+   queryScheduler:
+     replicas: 0
+   distributor:
+     replicas: 0
+   compactor:
+     replicas: 0
+   indexGateway:
+     replicas: 0
+   bloomCompactor:
+     replicas: 0
+   bloomGateway:
+     replicas: 0
+   EOF
+   ```
+
+2. Deploy Grafana Tempo to your cluster.
+
+   ```yaml
+   helm upgrade --install tempo tempo \
+   --repo https://grafana.github.io/helm-charts \
+   --version {{< reuse "docs/versions/otel-stack-tempo.md" >}} \
+   --namespace telemetry \
+   --create-namespace \
+   --values - <<EOF
+   persistence:
+     enabled: false
+   tempo:
+     receivers:
+       otlp:
+         protocols:
+           grpc:
+             endpoint: 0.0.0.0:4317
+   EOF
+   ```
+
+3. Verify that the Grafana pods are running. 
+   
+   ```sh
+   kubectl get pods -n telemetry -l 'app.kubernetes.io/name in (loki,tempo)'
+   ```
+   
+   Example output: 
+   ```console
+   NAME                   READY   STATUS    RESTARTS   AGE
+   loki-0                 2/2     Running   0          3m45s
+   loki-chunks-cache-0    2/2     Running   0          3m45s
+   loki-results-cache-0   2/2     Running   0          3m45s
+   tempo-0                1/1     Running   0          2m10s
+   ```
+
+## Step 2: Install the OTel Collector {#otel-collector}
+
+The OpenTelemetry collector acts as a centralized agent that scrapes metrics from the {{< reuse "/docs/snippets/kgateway.md" >}} control plane and data plane gateway proxies. Then, the OTel collector exposes these metrics in Prometheus format so that other tools in your observability stack, such as Grafana, can in turn scrape the OTel collector and visualize the data.
+
+By using an OTel collector to aggregate metrics, you avoid having to configure each application individually to send their metrics to each backend observability tool. This setup simplifies your setup, lets you more easily change backends, improves reliability and debuggability, and lets you optimize preprocessing activities such as filtering, transforming, or enriching the metrics before scraping.
+
+You can deploy three separate OTel collectors that are optimized for the three different types of telemetry data: metrics, logs, and traces. This way, you can scale and optimize each collector based on your telemetry needs.
+
+{{< callout type="warning" >}}
+The example pipelines in all three OTel collectors set up the `debug` exporter. This exporter is useful for testing and validation purposes. However, for production scenarios, remove this exporter to avoid performance impacts.
+{{< /callout >}}
+
+1. Deploy the metrics collector to handle numerical measurements and time-series data. Note that you can also use the `promexporter` endpoint with Prometheus to scrape metrics from the collector pod, if you prefer the `pull` model to the `push` model.
+
+   ```yaml
+   helm upgrade --install opentelemetry-collector-metrics opentelemetry-collector \
```

Comment:
Not sure if this is important, but got these warnings about resource limits when I installed. Should we set some in the settings here?

```
NOTES:
[WARNING] No resource limits or requests were set. Consider setting resource requests and limits for your collector(s) via the `resources` field.

[WARNING] "useGOMEMLIMIT" is enabled but memory limits have not been supplied so the GOMEMLIMIT env var could not be added. Solve this problem by setting resources.limits.memory or disabling useGOMEMLIMIT
```

Reply:
I wouldn't bother, otherwise people will take it as granted then complain why our examples are not working at scale.